git-multimail/git_multimail.py

```diff
@@ -3425,12 +3425,19 @@ def get_fromaddr(self, change=None):
 
 
 class GiteaEnvironmentHighPrecMixin(Environment):
+
+    def __init__(self, **kw):
+        super(GiteaEnvironmentHighPrecMixin, self).__init__(**kw)
+        self.COMPUTED_KEYS += ['repo_user_name']
+
     def get_pusher(self):
         return self.osenv.get('GITEA_PUSHER_NAME', 'unknown user')
 
     def get_pusher_email(self):
         return self.osenv.get('GITEA_PUSHER_EMAIL')
 
+    def get_repo_user_name(self):
+        return self.osenv.get('GITEA_REPO_USER_NAME')    
```

Comment:
One missing blank line here, and spurious trailing whitespaces, cf. the CI output:
```
/home/runner/work/git-multimail/git-multimail/t/../git-multimail/git_multimail.py:3440:54: W291 trailing whitespace
/home/runner/work/git-multimail/git-multimail/t/../git-multimail/git_multimail.py:3442:1: E302 expected 2 blank lines, found 1
```